platform/darwin/src/MGLShapeOfflineRegion.mm

```diff
@@ -22,6 +26,17 @@ @implementation MGLShapeOfflineRegion {
 
 @synthesize styleURL = _styleURL;
 
+#if TARGET_OS_IPHONE || TARGET_OS_SIMULATOR
```

Comment:
This needs to be inside the method, otherwise you'll be missing an implementation for `offlineStartEventAttributes`. You could either wrap the MMEEvents, or just have an else clause where you return a `@{}`